packages/server-wallet/src/wallet/__test__/integration/join-channel.test.ts

```diff
@@ -0,0 +1,68 @@
+import {simpleEthAllocation} from '@statechannels/wallet-core';
+
+import {Channel} from '../../../models/channel';
+import {Wallet} from '../..';
+import {seed} from '../../../db/seeds/1_signing_wallet_seeds';
+import {truncate} from '../../../db-admin/db-admin-connection';
+import knex from '../../../db/connection';
+import {stateWithHashSignedBy} from '../fixtures/states';
+import {bob} from '../fixtures/signing-wallets';
+import {channel} from '../../../models/__test__/fixtures/channel';
+
+let w: Wallet;
+beforeEach(async () => {
+  await truncate(knex);
+  w = new Wallet();
+});
+
+// Make sure alice's PK is in the DB
+beforeEach(async () => {
+  await seed(knex);
+});
```

Comment:
Any reason not to wrap all of this and the `it`s in a `describe`?

Reply:
Is there a reason to do that?